### DISCUSSION TITLE
Is v4 free now? Unable to access dashboard images/gallery.

### DISCUSSION THREAD

**Discussion by Jas-SinghFSU** [chevereto/chevereto#47] [Q&A] [answered]
I was planning to go to v4 from v3 since I heard that free users can access versions available a year after (I might have misunderstood). But when I go to the dashboard of v4, all the features (gallery, images, etc.) have a `$PRO` tag. Is this expected or did I configure my instance incorrectly?

**Comment by Jas-SinghFSU** [comment]
Scratch that, I seem to have found the answer https://rodolfoberrios.com/2022/12/01/chevereto-goes-free/ 

I suppose this is a good time to ask. Is Chevereto Pro a one time purchase or a recurring purchase? Will it include future updates (v5, v6, etc.)?

**Reply by rodber** [reply, marked as answer]
* For the lifespan of purchased major version

https://chevereto.com/pricing
